components/ItemCard/index.js

```diff
@@ -8,42 +8,19 @@ import CardImage from "./CardImage.js";
 import CardDescription from "./CardDescription.js";
 import UsernameButton from "./UsernameButton.js";
 
-const ItemCard = ({ product }) => {
-  console.log("🚀 ~ file: index.js ~ line 12 ~ ItemCard ~ product", product);
-  const [isFavorited, setFavorited] = useState(false);
-
-  const handleFavoritedItem = () => {
-    setFavorited(() => !isFavorited);
-  };
+const ItemCard = ({ children, product }) => {
 
   return (
-    <Card raised sx={{ width: "15vw", cursor: "pointer" }}>
-      <CardHeader
-        sx={{ zIndex: 9999 }}
-        action={
-          <IconButton
-            aria-label="add to favorites"
-            onClick={handleFavoritedItem}
-          >
-            {!isFavorited ? (
-              <FavoriteBorderIcon />
-            ) : (
-              <FavoriteIcon sx={{ color: "red" }} />
-            )}
-          </IconButton>
-        }
-      />
-      <CardImage />
-      <CardDescription />
-      <Link href={`/products/${product._id}`}>link</Link>
-      {product?.createdBy?.email}
-      <UsernameButton />
-      {/* {product?.createdBy && (
-        <Link href={`/users/${product?.createdBy?._id}`}>
-          {product?.createdBy?.email}
-        </Link>
-      )} */}
-    </Card>
+    <Link href={`/products/${product._id}`}>
+      <Card raised sx={{ width: "15vw", cursor: "pointer" }}>
```

Comment:
i think we should wrap the `Card` component in `CardActionArea` for semantic purposes

Reply:
in fact i strongly believe we should!

Reply:
DONE ITS SEXY